src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -36,3 +50,6 @@
         (#{ @comments.length})
```

Comment:
[unrelated to the PR] why is Rubocop not complaining about this space? :see_no_evil: